aikau/src/main/resources/alfresco/search/FacetFilter.js

```diff
@@ -187,14 +203,21 @@ define(["dojo/_base/declare",
        *
        * @instance
        */
-      onToggleFilter: function alfresco_search_FacetFilter__onToggleFilter(/*jshint unused:false*/ evt) {
-         if (this.applied)
+      onToggleFilter: function alfresco_search_FacetFilter__onToggleFilter(evt) {
+         if (evt.target !== this.labelNode && !this.fullWidthClick)
```

Comment:
Only the label node will support the click event when fullWidthClick is not true... so if fullWidthClick is not enabled then any other node but labelNode will have no impact. The unit tests verify the logic as well.